packages/tools/src/EditorFrame.tsx

```diff
@@ -358,6 +327,135 @@ export const EditorFrame = (props: EditorFrameProps) => {
   const s = props.classes;
   const domid = 'toolbar' + useDomid();
 
+  const base64ToBlob = (
```

Comment:
Move to separate function? Might want to add tests for this + there are ways for this to throw (e.g. `mimeString` for some reason does not include `/`) so it could be try-caught by caller.